docs/source/upcoming_release_notes/49-daq_d2scan.rst

```diff
@@ -0,0 +1,23 @@
+49 daq_d2scan
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- Add daq_d2scan
+- Add n-dimensional daq scans: daq_dnscan, daq_anscan
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- roberttk
```

Comment:
We've been making these github usernames by convention for proper call-outs in the tag
```suggestion
- tangkong
```

Reply:
I actually forgot that was my github username... thanks.